### PR TITLE
Deprecate functions in tests/utils.go

### PR DESCRIPTION
**Pull request by xpivarc**
**What this PR does / why we need it**:
These functions are deprecated by libvmi.

New tests should be using one of the convenient factory functions: `NewAlpine`,... or construct new 
with `New` function.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jcanocan** [comment]
What do you think about add a tip/example in the comment pointing out how to achieve the same with libvmi?

**Review by EdDev — APPROVED** [review]

File: tests/utils.go [hunk 1]
```diff
@@ -510,6 +513,9 @@ func NewRandomVirtualMachineInstanceWithDisk(imageUrl, namespace, sc string, acc
 	return NewRandomVMIWithDataVolume(dv.Name), dv
 }
 
+// New NewRandomVirtualMachineInstanceWithFileDisk
```
Comment (acardace) [inline tests/utils.go:516]: Double 'New'?

Comment (xpivarc) [reply tests/utils.go]: Thanks! Fixed.

**Review by dhiller — COMMENTED** [review]
Thank you for taking this into the right direction! :+1: 

/approve

/hold to fix the small double "New"

**Comment by oshoval** [comment]
Btw, maybe would be good to consider a follow PR that enforce no new deprecated functions are introduced in the diff ?
(there might be even a linter that does it, for example visual studio code + go plugin knows to mark deprecated funcs)

**Pull request merged**